packages/netsuite-adapter/src/service_url/file_cabinet.ts

```diff
@@ -14,32 +14,22 @@
 * limitations under the License.
 */
 
-import { CORE_ANNOTATIONS, InstanceElement } from '@salto-io/adapter-api'
-import { logger } from '@salto-io/logging'
-import { ServiceUrlSetter } from './types'
+import { InstanceElement, isInstanceElement } from '@salto-io/adapter-api'
 import { isFileCabinetInstance, isFileInstance } from '../types'
-import { INTERNAL_ID } from '../constants'
-
-const log = logger(module)
-
-const generateUrl = (element: InstanceElement): string | undefined => {
-  const id = element.value[INTERNAL_ID]
-  if (id === undefined) {
-    log.warn(`Did not find the internal id of ${element.elemID.getFullName()}`)
-    return undefined
-  }
+import { setElementsUrls } from './elements_urls'
+import { ServiceUrlSetter } from './types'
 
-  return isFileInstance(element)
+const generateUrl = (id: number, element: InstanceElement): string | undefined =>
+  (isFileInstance(element)
```

Comment:
a little bit nicer IMO
```suggestion
const generateUrl = (id: number, element: InstanceElement): string | undefined => (
  isFileInstance(element)
```